### PR TITLE
Update guide for validating metrics

### LINKED ISSUE #1553: Incorrect direction for validating metrics
## This is an issue with:

- [ ] Bug (site functionality or styling)
- [x] Errata (fix needed for doc content)
- [ ] New content (guide wanted)
- [ ] Update (Add missing or refresh existing content)
- [ ] Enhancement (add new site functionality)

## Expected Behavior
5.4.0 introduced a bugfix where metrics are not saved within the event: https://github.com/sensu/sensu-go/issues/2743.

## Current Behavior
States that metrics are saved within the event.
```
If the check output is formatted correctly according to its output_metric_format, the metrics will be extracted in Sensu Metric Format, and saved within the event. You should expect to see logged errors if Sensu is unable to parse the check output. You can validate that metrics have been extracted from your check through your handler, or through the resulting event. The example check we used would yield an event similar to the one below:
```

## Possible Solution
```
If the check output is formatted correctly according to its output_metric_format, the metrics will be extracted in Sensu Metric Format, and passed to the event pipeline. You should expect to see logged errors if Sensu is unable to parse the check output. You can validate that metrics have been extracted from your check through the resulting event passed to the handler. The example check we used would yield an event similar to the one below:
```

## Steps to Reproduce (for bugs)
https://docs.sensu.io/sensu-go/5.9/guides/extract-metrics-with-checks/#validating-the-metrics

## Context
5.4.0 change not reflected in past/current docs.

### PR DESCRIPTION
**Pull request by rgeniesse**
## Description
Updates extracting metrics guide to correct how to validate metrics.

## Motivation and Context
Closes #1553 

## Review Instructions
- [ ] Copy to all versions

### CODE REVIEW

**Review by cwjohnston — CHANGES_REQUESTED** [review]
In reviewing this PR it became clear to me that we need to document how to examine the event being sent to handlers.

I've opened https://github.com/sensu/sensu-docs/pull/1723 to add such an example to the troubleshooting guide. Let's get that merged an update this PR with a suggestion to validate the presence of metrics using a debug handler as shown there.

File: content/sensu-go/5.12/guides/extract-metrics-with-checks.md [hunk 1]
```diff
@@ -74,11 +74,10 @@ example              | {{< highlight string >}}sys.cpu.user 1356998400 42.5 host
...
 
 If the check output is formatted correctly according to its `output_metric_format`,
-the metrics will be extracted in Sensu Metric Format, and saved within the
-event. You should expect to see logged errors if Sensu is unable to parse
-the check output. You can validate that metrics have been extracted from your
-check through your handler, or through the resulting event. The example check
-we used would yield an event similar to the one below:
+the metrics will be extracted in Sensu Metric Format, and passed to the event pipeline.
+You should expect to see logged errors if Sensu is unable to parse the check output.
+You can validate that metrics have been extracted from your check through the resulting event passed to the handler.
```
Comment (cwjohnston) [inline content/sensu-go/5.12/guides/extract-metrics-with-checks.md]:
```suggestion
You can validate that metrics have been extracted from your check by inspecting the event passed to the handler. See our troubleshooting guide for an [example debug handler][10] which writes events to a file for inspection.
```

Comment (cwjohnston) [reply content/sensu-go/5.12/guides/extract-metrics-with-checks.md]: TODO: add appropriate troubleshooting anchor link to the bottom of the document as `[10]:`

**Review by cwjohnston — CHANGES_REQUESTED** [review]

**Review by cwjohnston — APPROVED** [review]

**Comment by rgeniesse** [comment]
Only changed starting at 5.4.0 as that is when metrics were not persisted in etcd, so users on 5.3.0 and before can check the event output via sensuctl to see if metrics are making it that far.

https://docs.sensu.io/sensu-go/5.13/release-notes/#5-4-0-release-notes

**Pull request merged**